ConvivaTestApp/src/androidTest/java/com/bitmovin/analytics/conviva/testapp/PlaybackControlsTrackingTests.kt

```diff
@@ -117,6 +114,7 @@ class PlaybackControlsTrackingTests: TestBase() {
         activityScenario.onActivity { activity: MainActivity ->
             verifyOrder {
                 videoAnalyticsMock?.reportPlaybackMetric(ConvivaSdkConstants.PLAYBACK.SEEK_STARTED,-1)
+                videoAnalyticsMock?.reportPlaybackMetric(ConvivaSdkConstants.PLAYBACK.PLAYER_STATE, ConvivaSdkConstants.PlayerState.BUFFERING)
```

Comment:
This test should also check for PLAYING state after SEEK_ENDED. Also please add a new test case like `liveTimeshiftWhenPaused` to test same scenario when timeshift is done in paused state.

Reply:
Oh, I need to add this one too, sorry.

Reply:
Just added this. Think that's it now then.